inputs/org.osbuild.containers

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/python3
+"""Inputs for container images
+
+This reads images from the `org.osbuild.container` directory in the
+sources store.
+
+The store is indexed by the "container image id", which is the digest
+of the container configuration file (rather than the outer manifest)
+and is what will be shown in the "podman images" output when the image
+is installed. This digest is stable as opposed to the manifest digest
+which can change during transfer and storage due to
+e.g. recompression.
+
+"""
+
+import os
+import sys
+import json
+
+from osbuild import inputs
+
+
+SCHEMA = r"""
+"additionalProperties": false,
+"required": ["type", "origin", "references"],
+"properties": {
+  "type": {
+    "enum": ["org.osbuild.containers"]
+  },
+  "origin": {
+    "description": "The origin of the input (source)",
+    "type": "string",
+    "enum": ["org.osbuild.source", "org.osbuild.source"]
+  },
+  "references": {
+    "description": "Container image id",
+    "oneOf": [{
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    }, {
+      "type": "object",
+      "additionalProperties": false,
+      "patternProperties": {
+        ".*": {
+          "type": "object",
+          "additionalProperties": true,
+          "properties": {
+          }
+        }
+      }
+    }]
+  }
+}
+"""
+
+class ContainersInput(inputs.InputService):
+
+    def map(self, store, origin, refs, target, _options):
+        source = store.source("org.osbuild.container")
```

Comment:
Not a big deal, but maybe should match the input name, i.e. `containers`.
```suggestion
        source = store.source("org.osbuild.containers")
```

Reply:
Thats not what i got from https://github.com/osbuild/osbuild/pull/952#pullrequestreview-874407693 ?

Reply:
Ah, sorry, mea culpa. :/ I seem to have typo-ed that.